x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -113,6 +119,24 @@ export const CredentialsList: React.FC = () => {
     hidePerPageOptions: true,
   };
 
+  if (items.length < 1) {
+    return (
+      <EuiEmptyPrompt
+        iconType="editorStrike"
+        title={
+          <h2>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.credentials.empty.title', {
+              defaultMessage: 'No API Keys have been created yet.',
+            })}
+          </h2>
+        }
+        body={i18n.translate('xpack.enterpriseSearch.appSearch.credentials.empty.body', {
+          defaultMessage: 'Click Create a key to make your first one.',
```

Comment:
Minor copy suggestion:

```suggestion
          defaultMessage: 'Click the “Create a key” button to make your first one.',
```

Although part of me wonders if we shouldn't just straight up put a "Create a key" button in here for more straightforward UX? 🤔 I might be overthinking it haha

Reply:
This was the UI from the original App Search. It had the same message, and hid the button above, and replaced it with a button below.

It just seemed like code we didn't need. Why take the time to move the button when it already exists above?

That was my thought anyway. I probably should have kept it just for consistency between the old UI and this UI.

![screencapture-localhost-3002-as-2020-10-15-09_11_01](https://user-images.githubusercontent.com/1427475/96131539-4f7ffa00-0ec7-11eb-8973-6149560881e4.png)
![screencapture-localhost-3002-as-2020-10-15-09_11_10](https://user-images.githubusercontent.com/1427475/96131550-53ac1780-0ec7-11eb-97a7-6d90f54218d0.png)

Reply:
I'm good with not moving the button and leaving the empty prompt as is FWIW! +1 to "why take the time to move the button". Just the copy tweak would be fine by me 👍

Reply:
https://github.com/elastic/kibana/pull/80034/commits/d52131d1eb8e5722702c9c3b7b4717b2d71ae572